uzipoc-cibg/Issues/uzi-2/description.xml

```diff
@@ -0,0 +1,114 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Uzi register</en>
+		<nl>Uzi register</nl>
+	</Name>
+	<ShortName>
+		<en>Uzi register</en>
+		<nl>Uzi register</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>uzipoc-cibg</IssuerID>
+	<CredentialID>uzi-2</CredentialID>
+	<Description>
+		<en>Uzi register attributen</en>
+		<nl>Uzi register gegevens</nl>
+	</Description>
+	<IssueURL>
+		<en>https://irma-credentials.uzipoc.nl</en>
+		<nl>https://irma-credentials.uzipoc.nl</nl>
+	</IssueURL>
+
+	<Attributes>
+		<Attribute id="initials">
+			<Name>
+				<en>Initials</en>
+				<nl>Initialen</nl>
+			</Name>
+			<Description>
+				<en>Your initials</en>
+				<nl>Uw initialen</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="surname_prefix">
+			<Name>
+				<en>Surname prefix</en>
+				<nl>Tussenvoegsel</nl>
+			</Name>
+			<Description>
+				<en>Your surname prefix</en>
+				<nl>Uw tussenvoegsel</nl>
+			</Description>
+		</Attribute>
```

Comment:
I'm sorry that I only notice this now, but normally we don't accept underscores in attribute names. Normally, either flatcase or lowerCamelCase is used here. The only special character that we sometimes use is a dash (`-`). This is not documented very well in our documentation.